subtrack.Tests/SubscriptionCalculatorTests/IGetDueDaysTestData.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace subtrack.Tests.SubscriptionCalculatorTests
+{
+    public interface IGetDueDaysTestData
```

Comment:
Is this file used somewhere? (Might be blind cuz on phone)

Reply:
auto complete probably